### ISSUE TITLE
RC test: Wave Interference 1.0.0-rc.1

### ISSUE THREAD

**Issue by samreid** [phetsims/QA#274]
@ariel-phet, @KatieWoe, wave-interference/1.0.0-rc.1 is ready for RC testing. We would like to publish this on the main site by Jan 31, and this will be shared with Google.org.  Document issues in https://github.com/phetsims/wave-interference/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works. (no sound in this sim yet)
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR if it not there.
- If this is rc.1 please do a memory test.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/wave-interference/1.0.0-rc.1/phet/wave-interference_en_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/wave-interference/1.0.0-rc.1/phet/wave-interference_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/wave-interference/1.0.0-rc.1/phet/xhtml/wave-interference_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1wgAKZmWknpMONzv6sW5KcrfdTkB2q8Q0GCIGwDGUIUs/edit#gid=1218896198)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by ariel-phet** [comment]
@KatieWoe very much top priority along with #273

Please note - given the tight timeline AND the fact that we will continue working on this sim and unquestionably publishing a 1.1 version, please test in more of a "normal use case mode". Basically try to make sure there are no showstopper bugs on any major platform. Even if this sim does not have full RC coverage, we are going to publish on Thursday. So please be judicious with each platform. 

I don't feel the dev testing was quite as heavy on this sim as EFAC with some recent changes, so potentially push a little harder on a few major platforms (chromebook, newer ipad, Win chrome, Mac safari). But again, really looking for things fixable by Thursday and a reasonable confidence the sim is in decent shape for a 1.0 release. 

In short, you will need to test both this sim and EFAC in parallel on Tuesday so if there is something minor it can be patched up for a RC.2 spot check on Wednesday or Thursday morning.

**Comment by KatieWoe** [comment]
- [ ] Win 10 chrome
- [ ] Win 10 firefox
- [ ] Win 10 edge
---
- [ ] Mac 10.14 safari
- [ ] Mac 10.14 firefox
- [ ] Mac 10.14 chrome
---
- [ ] chrome os
- [ ] iOS 12
- [ ] iOS 10
---
If all these are done (on both this and EFAC) and there is still time, please test some of the other platforms at your discretion.

**Comment by KatieWoe** [comment]
QA is done with the main platforms.  I also did a fair amount of testing on IE and all looked OK.  @ariel-phet if he wants anything more done before this is officially called to an end.

**Comment by ariel-phet** [comment]
I think we can call it. Over to @samreid

**Comment by samreid** [comment]
Thanks for the testing!  Everything is tracked is other issues, closing.
